client/src/components/icons/DeleteIcon.tsx

```diff
@@ -0,0 +1,26 @@
+import {TouchableOpacity, View} from "react-native";
+import {MaterialCommunityIcons} from "@expo/vector-icons";
+import {IconsStyles} from "./FavoriteIcon";
+import React from "react";
+
+interface Props {
+    itemId: string;
+    backgroundColor?: string;
+    onDeletePress?: () => void;
+}
+
+export const DeleteIcon = ({onDeletePress, backgroundColor = '#f2f2f2'}: Props) => {
```

Comment:
DeleteIcon, EditIcon, FavoriteIcon seem VERY similar
I might reuse all the three with a generic icon component that receives `style` for the View, `handleOnPress`, `backgroundColor = '#f2f2f2'`, and also `icon` which is an object containing: `name`, `color`, `size`.

Then DeleteIcon, EditIcon, FavoriteIcon will use this generic component with the relevant parameters

Reply:
done